src/settingsholder.cpp

```diff
@@ -271,6 +274,10 @@ GETSETDEFAULT(SETTINGS_DEVELOPERUNLOCK_DEFAULT, bool, toBool,
 GETSETDEFAULT(SETTINGS_STAGINGSERVER_DEFAULT, bool, toBool,
               SETTINGS_STAGINGSERVER, hasStagingServer, stagingServer,
               setStagingServer, stagingServerChanged)
+GETSETDEFAULT(SETTINGS_STAGINGSERVERADDRESS_DEFAULT, QString, toString,
```

Comment:
Instead of having a default value == "", what about if you do:
```
GETSETDEFAULT(getEnvVariable("MVPN_API_BASE_URL"), ...
```

where `getEnvVariable` is something _private_ like:

```
QString Settings::getEnvVariable(const QString& name) const {
  QProcessEnvironment pe = QProcessEnvironment::systemEnvironment();
  if (pe.contains(name)) return pe.value(name);
  return QString();
}
```

Reply:
Shouldn't this default to `Constants::API_STAGING_URL` when the environment variable is set? Otherwise it seems like the only way to test against our normal staging server is by typing the URL in manually after each reset of the settings.

Reply:
This is one of the goals of this. We do not want it to be as easy as it currently is for a user to accidentally find the dev menu and then switch to the staging server. 
Copy pasting the address into this field is fairly easy and not a big problem if you know the address.
We could look into ways to set this at build time for easier development.